cmd/invoker/main.go

```diff
@@ -0,0 +1,228 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+)
+
+type LambdaEvent struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+	URL      string `json:"url"`
+	Debug    bool   `json:"debug"`
+	Reload   bool   `json:"reload"`
+	Delay    int    `json:"delay"`
+	JoinTeam string `json:"join_team,omitempty"`
+}
+
+type LambdaResponse struct {
+	StatusCode int               `json:"statusCode"`
+	Headers    map[string]string `json:"headers"`
+	Body       string            `json:"body"`
+}
+
+type ResponseBody struct {
+	Message    string `json:"message"`
+	Screenshot string `json:"screenshot,omitempty"`
+	Error      string `json:"error,omitempty"`
+}
+
+func main() {
+	// Define command line flags
+	username := flag.String("username", "", "Username for login (with number suffix after '-')")
+	password := flag.String("password", "", "Password for login")
+	url := flag.String("url", "", "URL of the Mattermost instance")
+	debug := flag.Bool("debug", false, "Enable debug mode (includes screenshot)")
+	reload := flag.Bool("reload", false, "Enable reload after delay")
+	delay := flag.Int("delay", 5000, "Delay in milliseconds after login")
+	joinTeam := flag.String("join_team", "", "Team name to join")
+	functionName := flag.String("function", "mattermost-login", "Name of the Lambda function")
+	region := flag.String("region", "us-east-1", "AWS region")
+	outputFile := flag.String("output", "", "File to save screenshot (if debug is enabled)")
+	count := flag.Int("count", 1, "Number of concurrent Lambda invocations")
+
+	flag.Parse()
+
+	// Validate required parameters
+	if *username == "" || *password == "" || *url == "" {
```

Comment:
Any reason for using pointers?

Reply:
That's just what the flag package returns.

Reply:
I always use the `Var` variant. Example -> https://github.com/mattermost/mattermost-plugin-calls/blob/0aefee7b5827224b5c4a1ab8f517a368d97fe1c2/lt/cmd/lt/main.go#L70-L71

Reply:
Ah yes, that also works. Will leave it at this.